docker-compose/camunda-8.6/docker-compose-run.yaml

```diff
@@ -0,0 +1,110 @@
+# While the Docker images themselves are supported for production usage,
+# this docker-compose.yaml is designed to be used by developers to run
+# an environment locally. It is not designed to be used in production.
+# We recommend to use Kubernetes in production with our Helm Charts:
+# https://docs.camunda.io/docs/self-managed/platform-deployment/kubernetes-helm/
+# For local development, we recommend using KIND instead of `docker-compose`:
+# https://docs.camunda.io/docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster/
+
+# This is a lightweight configuration with Zeebe, Operate, Tasklist, and Elasticsearch
+# See docker-compose.yml for a configuration that also includes Optimize, Identity, and Keycloak.
+
+services:
+  camunda: # https://docs.camunda.io/docs/self-managed/platform-deployment/docker/#zeebe
+    image: camunda/camunda:${CAMUNDA_PLATFORM_RUN}
+    container_name: camunda
+    ports:
+      - "26500:26500"
+      - "9600:9600"
+      - "8080:8080"
```

Comment:
Question: should we map this to 8080?

This is currently the port c8run uses, but it would clash with a locally running spring boot app. Therefore, we use 8081+ in the other docker-compose files and the helm charts.